### PR TITLE
Add Devise

### PR DESCRIPTION
**Pull request by ppmarq1**
Project Requirements

- Remove current_user method in ApplicationController.
- [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
   - Can register a new user.
   - User logs in with a combination of email and password.
   -  password should be stored in the database.
   - Ask for confirmation of email.
   - Can reset password.
- [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.

### CODE REVIEW

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Stanley,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby a linter check is failing. Kindly ensure that **all** checks are passing.
image


 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by stanleySimeon** [comment]
Hello @


*rloterh,How are you?Kindly can you tell me the reason that the request is
invalid?*

**Comment by stanleySimeon** [comment]
@rloterh, kindly git GitGuardian Security checks is not part of linters
and the project requirements

File: app/models/user.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Create user model
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```
Comment (dasileker) [inline app/models/user.rb]: - [x] Hello nice work but you have missed one piece that is `:confirmable`.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @stanleySimeon,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- The codes are clean 
- The Pr is very nice .
- The readme file is good.

## Required Changes ♻️
_Check the comments under the review._

- **Nice job on the device but you have missed some requirements also please provide details about how to add the `tailwind` in rails, and please please deactivate the `gitgardian` so the reviewers don't get confused about the linters.** 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (stanleySimeon) [reply app/models/user.rb]: Thanks a lot!
Resolved

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @ppmarq1 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Let me pont out:
- The current_user method in ApplicationController has been removed 👍🏻 
- Devise has been setup properly 👍🏻 
- A user can log in with a combination of email and password 👍🏻 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rloterh** [comment]
> @rloterh, kindly git GitGuardian Security checks is not part of linters and the project requirements

Hello @stanleySimeon 
All checks needs to pass, therefore kindly ensure to remove any redundant linter(s).

Cheers!

**Pull request merged**